src/codegen/generate-classes.ts

```diff
@@ -1927,7 +1939,7 @@ void GlobalObject::visitGeneratedLazyClasses(GlobalObject *thisObject, Visitor&
 const ZIG_GENERATED_CLASSES_HEADER = `
 /// Generated code! To regenerate, run:
 ///
-///    make codegen
+///    bun run build
 ///
 /// This file is generated by:
 ///  1. \`bun src/bun.js/scripts/generate-classes.ts\`
```

Comment:
since we're editing this comment, we should fix the entire comment

```suggestion
///  1. \`bun src/codegen/generate-classes.ts\`
```